pages/docs/couple-your-code/couple-your-code-porting-v2-3.md

```diff
@@ -113,39 +113,46 @@ Please add breaking changes here when merged to the `develop` branch.
   - Where declaring a preCICE object, replace the `precice::SolverInterface` type with `precice::Participant`.
   - Where constructing a preCICE object, replace the `precice::SolverInterface( ... )` constructor with `precice::Participant( ... )`.
   - Consider renaming your objects from, e.g., `interface` to `participant`, to better reflect the purpose and to be consistent with the rest of the changes.
+- Steering methods
+  - Replace `double preciceDt = initialize()` and `double preciceDt = advance(dt)` with `initialize()` and `advance(dt)`, as they don't have a return value.
+  - Use `double preciceDt = getMaxTimeStepSize()`, where you need the max time step size or the relative end of the time window.
+- Dimensions
+  - Replace `getDimensions()` with `getMeshDimensions(meshName)`
+  - Replace custom logic to determine the dimensionality of data with `getDataDimensions(meshName, dataName)`
+- Migrate to mesh and data ids to names
+  - Remove the now obsolete calls to `getMeshID()` and `getDataID()` and uses of types `MeshID` and `DataID`.
```

Comment:
Which uses of types `MeshID` and `DataID` do you mean here? It is not immediately clear.

Reply:
I mean variable declarations and things like that.

Reply:
Okay. Perhaps being more explicit in the wording would help, or also just saying *... which means data and mesh IDs are no longer used in the API.*

Reply:
```suggestion
  - Remove the now obsolete calls to `getMeshID()` and `getDataID()`.
  - Remove uses of the now obsolete typedefs `MeshID` and `DataID`.
```

Like that?

Reply:
Were they typedefs? Users (me) probably just saw them as ints..